crates/anvil/src/hardfork.rs

```diff
@@ -60,22 +61,23 @@ impl EthereumHardfork {
     /// Get the first block number of the hardfork.
     pub fn fork_block(&self) -> u64 {
```

Comment:
I think this is basically, just:

https://github.com/alloy-rs/hardforks/blob/ff055c0553ad179e00070eea93434958049bb814/crates/hardforks/src/hardfork/ethereum.rs#L53

?